### PR TITLE
incorporate changes from https://github.com/demisto/content/pull/4858…

### PR DESCRIPTION
**Pull request by avidan-H**
…/files for update_id_set to support packs

### CODE REVIEW

**Comment by yardensade** [comment]
Seems like you will have to address the changes in test file paths to absolute ones, take a look in my pr or at other validator tests who use files:
https://github.com/demisto/demisto-sdk/pull/15

**Comment by avidan-H** [comment]
hm... not quite the same cause the `update_id_set_test.py` file was actually testing against the real fetched pack data from the content repo... I could drop the cortexxdr pack from the content repo into test data here and then change the path to that. little bit pita but whatever, unless you have a better idea

**Comment by yardensade** [comment]
You can attach the pack as test data, and use it like that with minor changes to file paths

File: demisto_sdk/common/scripts/update_id_set.py [hunk 1]
```diff
@@ -1,19 +1,35 @@
...
+    PACKS_INTEGRATION_REGEX,
+    # Scripts
+    SCRIPT_REGEX,
+    PACKS_SCRIPT_YML_REGEX,
+    # Playbooks
+    PLAYBOOK_REGEX,
+    TEST_PLAYBOOK_REGEX,
+    PACKS_PLAYBOOK_YML_REGEX,
+    PACKS_TEST_PLAYBOOKS_REGEX
+)
```
Comment (liorblob) [inline demisto_sdk/common/scripts/update_id_set.py]: Shouldn't this be consts?

File: demisto_sdk/common/scripts/update_id_set.py [hunk 2]
```diff
@@ -309,13 +332,68 @@ def process_testplaybook_path(file_path):
...
+
     return playbook, script
 
 
+def get_integrations_paths():
+    path_list = [
+        ['Integrations', '*'],
+        ['Beta_Integrations', '*'],
+        ['Packs', '*', 'Integrations', '*']
+    ]
```
Comment (liorblob) [inline demisto_sdk/common/scripts/update_id_set.py]: CONSTS DUDE

**Comment by avidan-H** [comment]
> and make the tests pass

lol thank you for the illuminating comment

**Comment by avidan-H** [comment]
> You can attach the pack as test data, and use it like that with minor changes to file paths

isn't that what I said? Or are you just agreeing with me?

File: demisto_sdk/common/scripts/update_id_set.py [hunk 3]
```diff
@@ -1,19 +1,35 @@
...
+import itertools
 import os
 import glob
 import json
 from collections import OrderedDict
 from multiprocessing import Pool, cpu_count
+from distutils.version import LooseVersion
 import time
 
 # TODO: do not import *
```
Comment (Itay4) [inline demisto_sdk/common/scripts/update_id_set.py]: hessing what is this ?

File: demisto_sdk/common/scripts/update_id_set.py [hunk 4]
```diff
@@ -1,19 +1,35 @@
+# !/usr/bin/env python
```
Comment (Itay4) [inline demisto_sdk/common/scripts/update_id_set.py]: isnt the code python3?

**Comment by yardensade** [comment]
> > You can attach the pack as test data, and use it like that with minor changes to file paths
> 
> isn't that what I said? Or are you just agreeing with me?

As test data, not as content, all I'm saying for matter of clarity.

File: demisto_sdk/common/scripts/update_id_set.py (hunk 3 above)
Comment (avidan-H) [reply demisto_sdk/common/scripts/update_id_set.py]: everything you see here has been copied wholesale from the PR linked in the PR title above so I have no idea

**Pull request merged**